packages/artifacts/src/download/download.browser.ts

```diff
@@ -0,0 +1,28 @@
+import { type Project, projects } from '../projects'
+import type { SnarkArtifacts, Version } from './types'
+
+export default async function maybeGetSnarkArtifacts(
+  project: Project,
+  options: {
+    parameters?: (bigint | number | string)[]
+    version?: Version
+    cdnUrl?: string
+  } = {},
+): Promise<SnarkArtifacts> {
+  if (!projects.includes(project)) {
+    throw new Error(`Project '${project}' is not supported`)
+  }
+
+  options.version ??= 'latest'
+  options.cdnUrl ??= 'https://unpkg.com'
+
+  const BASE_URL = `${options.cdnUrl}/@zk-kit/${project}-artifacts@${options.version}`
+  const parameters = options.parameters
+    ? `-${options.parameters.join('-')}`
+    : ''
+
+  return {
+    wasm: `${BASE_URL}/${project}${parameters}.wasm`,
+    zkey: `${BASE_URL}/${project}${parameters}.zkey`,
+  }
+}
```

Comment:
different from `@zk-kit/utils`
here we were re exporting `Project` and `projects`.
Not sure why?
Is it so that they are available when using [this](https://github.com/privacy-scaling-explorations/zk-kit/blob/66960e7e037cb657f745e50de30c6843ba20e5a4/packages/utils/src/snark-artifacts/snark-artifacts.browser.ts#L28) export

Reply:
> Is it so that they are available when using this export?

Yes, that's the reason!